build.gradle.kts

```diff
@@ -23,14 +23,15 @@ sourceSets {
             // We need the entire source directory here, otherwise we get a
             // "package is empty or does not exist" error during compilation.
             srcDir("src/main/java")
+            include("org.jetbrains.annotations")
```

Comment:
Why is this necessary?

Reply:
Hi, sorry to not get back to you right away! 

Honestly, I am not entirely sure this was necessary. I wasn't able to get the library to build without including this line; specifically the line `require annotations;` in `module-info.java` would fail. My knowledge on java 9 modules is very limited, so again I'm not sure this is necessary...